tools/depends/target/mariadb/02-symbol_rename-gnutls_clash.patch

```diff
@@ -0,0 +1,97 @@
+--- a/libmariadb/ma_hash.c
++++ b/libmariadb/ma_hash.c
+@@ -69,7 +69,7 @@
+ }
+ 
+ 
+-void hash_free(HASH *hash)
++void hash_free1(HASH *hash)
+ {
+   if (hash->free)
+   {
+@@ -287,7 +287,7 @@
+ 
+ 	/* Write a hash-key to the hash-index */
+ 
+-my_bool hash_insert(HASH *info,const uchar *record)
++my_bool hash_insert1(HASH *info,const uchar *record)
+ {
+   int flag;
+   uint halfbuff,hash_nr,first_index,idx;
+@@ -421,7 +421,7 @@
+ ** if there is a free-function it's called for record if found
+ ******************************************************************************/
+ 
+-my_bool hash_delete(HASH *hash,uchar *record)
++my_bool hash_delete1(HASH *hash,uchar *record)
```

Comment:
Prefix the new name of the function with the library name instead of appending 1.

Reply:
done as requested. Used a define to minimise the patch as well. Hope that was what you meant in your other comment.